### ISSUE TITLE
"Open New Tabs by Default" fails

### ISSUE THREAD

**Issue by worksong** [tabkit/tabkit2#22]
What steps will reproduce the problem?
1. address bar, search bar & bookmarks all open in active tab, despite "new tab" being checked.  Deselect, reboot, select, reboot doesn't help.

What version of the product are you using? On what operating system?
======Environment======
Firefox Version (if not latest stable version):  20.0.1 [but problem manifest will on v. 19]

Operating System (if not Windows):  Win 7 pro

**Comment by PikachuEXE** [comment]
Does this issue still exist with FF 45.x & latest version of Tab Kit?

**Comment by worksong** [comment]
Can't say because I've been rejecting FF upgrades due to Tab Kit incompatibility.  Wholesale failure to group, color, etc. started after 44.0.2.  I've tested every rehearse since then, including 46.0, with no success.  I've rolled all my PCs back to v. 44...


via mobile



-------- Original Message --------

**Comment by PikachuEXE** [comment]
Try using ESR version
That's more stable than the "normal" one

**Comment by worksong** [comment]
Tnx.  Traveling, but will try next week.


via mobile



-------- Original Message --------

**Comment by worksong** [comment]
Immediately tried latest FF [46.0.1] on my return.  Unlike every past version I’ve tried, it did NOT install with minimum functionality [e.g. no tab color, external links opening new window i/o new tab, bookmarks overwriting current tab].  Instead, FF declared TabKit incompatible & disabled it, with no option to re-enable in this version.  Have never seen that happen.

 

Before I could reinstall FF 44.0.2—the last TabKit-compatible version I found—I had to reboot for other reasons.  I was shocked when FF opened up with TabKit both active &, apparently, fully functional.  So far, it’s stayed that way through 2 more reboots, & highlighting, bookmarks, external links & everything else appear to be working normally.

 

Will advise if any change.  Also, will be at location with 2 more machines next weekend, & will try upgrading those to FF 46 as well.

 

Continued thanks for all your efforts.
